### PR TITLE
User story 30

### PR DESCRIPTION
**Pull request by leiyakenney**
- Adds pending_orders method to merchant (Ruby-heavy, but my ActiveRecord isn't the strongest, so we can refactor that later)
- Creates tests for story 30
- Adds Dunder Mifflin as a seed
- RSpec runs clean
- SimpleCov is at 99.71%

### CODE REVIEW

**Comment by leiyakenney** [comment]
There were a couple merge conflicts with the routes and something else small - I resolved them so whoever does the merge doesn't have to!

File: app/views/merchant/dashboard/index.html.erb [hunk 1]
```diff
@@ -5,7 +5,32 @@
...
+<h1 align = "center">Pending Orders</h1>
+<center>
+  <table>
+    <tr>
+      <th>Order ID</th>
+      <th>Order Date</th>
+      <th>Quantity of Items</th>
+      <th>Order Total</th>
+    </tr>
+    <% unless @merchant.no_orders? %>
```
Comment (nancylee713) [inline app/views/merchant/dashboard/index.html.erb]: Is this instance method defined in merchant model?

**Comment by nancylee713** [comment]
Thanks for finding the crucial links among various tables... now every model has access to each other's model as well as dependent models!

File: app/views/merchant/dashboard/index.html.erb (hunk 1 above)
Comment (leiyakenney) [reply app/views/merchant/dashboard/index.html.erb]: It's defined in the merchant/dashboard_controller. Is that okay? I have @merchant_user, @merchant, and @pending_orders defined there.

File: config/routes.rb [hunk 2]
```diff
@@ -3,8 +3,13 @@
 
   root to: "welcome#index"
 
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+
   namespace :merchant do
     get "/", to: "dashboard#index", as: :user
+    get "/orders/:id", to: "dashboard#show", as: :orders_show
```
Comment (tschaffer1618) [inline config/routes.rb]: Is this for showing a single order (like the link off of the dashboard)? If so, maybe :order_show might be more appropriate?

Comment (leiyakenney) [reply config/routes.rb]: This one is for all the pending orders, so that's why I made it plural. Happy to switch it to order if that's better!

**Pull request merged**